src/main/java/com/bakdata/kserve/client/KServeClient.java

```diff
@@ -1,7 +1,7 @@
 /*
  * MIT License
  *
- * Copyright (c) 2022 bakdata
+ * Copyright (c) 2024 bakdata GmbH
```

Comment:
Or copyright config seems to be inconsistent. Sometimes we use GmbH, sometimes not...